app/docs/0.11.x/plugin-development/custom-logic.md

```diff
@@ -171,18 +185,59 @@ return CustomHandler
 ### Plugins execution order
 
 <div class="alert alert-warning">
-  <strong>Note:</strong> This is still a work-in-progress API. For thoughts on how plugins execution order should be configurable in the future, see <a href="https://github.com/Mashape/kong/issues/267">Mashape/kong#267</a>.
+  <strong>Note:</strong> This is still a work-in-progress API. For thoughts on
+  how plugins execution order should be configurable in the future, see
+  <a href="https://github.com/Mashape/kong/issues/267">Mashape/kong#267</a>.
 </div>
 
-Some plugins might depend on the execution of others to perform some operations. For example, plugins relying on the identity of the consumer have to run **after** authentication plugins. Considering this, Kong defines **priorities** between plugins execution to ensure that order is respected.
+Some plugins might depend on the execution of others to perform some
+operations. For example, plugins relying on the identity of the consumer have
+to run **after** authentication plugins. Considering this, Kong defines
+**priorities** between plugins execution to ensure that order is respected.
 
-Your plugin's priority can be configured via a property accepting a number in the returned handler table:
+Your plugin's priority can be configured via a property accepting a number in
+the returned handler table:
 
 ```lua
 CustomHandler.PRIORITY = 10
 ```
 
-The higher the priority, the sooner your plugin's phases will be executed in regard to other plugins' phases (such as `:access()`, `:log()`, etc...). The current authentication plugins have a priority of `1000`.
+The higher the priority, the sooner your plugin's phases will be executed in
+regard to other plugins' phases (such as `:access()`, `:log()`, etc...).
+
+The current order of execution for the bundled plugins is:
+
+Plugin                    | Priority
+-------------------------:|:------------
+bot-detection             |
+cors                      |
+jwt                       |
+basic-auth                |
+oauth2                    |
+key-auth                  |
+hmac-auth                 |
+ldap-auth                 |
+ip-restriction            |
+acl                       |
+request-size-limiting     |
+rate-limiting             |
+response-ratelimiting     |
+response-transformer      |
+request-transformer       |
+aws-lambda                |
+datadog                   |
+udp-log                   |
+loggly                    |
+syslog                    |
+statsd                    |
+runscope                  |
+http-log                  |
+request-termination       |
+tcp-log                   |
+galileo                   |
+correlation-id            |
+file-log                  |
```

Comment:
need to add those priorities now that @hishamhm submitted #2812 as we discussed!